### ISSUE TITLE
Quest 2: Gather Remains and Side Quest: Confirmation Letter

### ISSUE THREAD

**Issue by Kalinter** [globemallow-mw/ossuaries-of-the-ghostfence#16]

**Comment by JoanyMcKarelyn** [comment]
Brief summary for Quest 2 Gather Remains: Once the Pilgrimage of the Dead is complete, the player is directed to the leader of the Order of Ghosts. The player gets a list of the recently deceased from the leader of the Order, aka Adansa Kren, and has to go collect remains in Ald-ruhn, Vivec, Gnisis, and then send an elderly NPC in Maar Gan a letter confirming their registration with the temple's remains donation program.

**Comment by JoanyMcKarelyn** [comment]
Quest ID: GG_Quest_2 (Follow Kalinter's naming)

Quest Name: TBD

**Comment by JoanyMcKarelyn** [comment]
Quest ID: GG_Comfirmation

Quest Name: Confirmation Letter Delivery

**Comment by JoanyMcKarelyn** [comment]
# BOOK 

## Remains Donation Registration (GG_bk_letter_unsigned)

TRIBUNAL TEMPLE
REMAINS DONATION - REGISTRATION FORM
Name: Musa Yahaz
Address: Gnisis
Gender: Female
To Tribunal Temple, 
I desire that, after my death, my remains be donated for the purpose of the sustain the Great Ghostfence.
I expect that the person / persons lawfully in-charge of my body after my death shall respect my wish. 
Yours sincerely,
(Donor's signature and full name)

# Topics

## duties 

Good, %PCRank %PCName. We need someone to collect some of the remains of the recently deceased. Get the details from my assistant Airen Dara. When it's done, report back to me.

We need someone to collect some of the remains of the recently deceased. Get the details from my assistant Airen Dara. When it's done, report back to me.

## recently deceased

Ask around in Gnisis where she is. She should be an elderly Dunmer lady according to the logs here. Return the document back to me once it is signed.

Technically, no. But, anyway, ask around in Gnisis where she is. She should be an elderly Dunmer lady according to the logs here. Return the document back to me once it is signed.

# Greeting

## Greeting 5

Condition: ActorID = GG_adansa_kren

What brings you to the Hall of Ghosts, %PCRank? Have you been lax in your duties?

Condition: ActorID = GG_airen_dara

Nice to meet you, %PCRank %PCName. I am Adansa's assistant Airen Dara. I handle most of the documents, reports, logs, and reminders here. Are you here for the new duty assignment?

Player->AddItem "GG_bk_recently_deceased" 1
Choice "Yes, The Commander ask me to collect the remains and I am supposed to get the details from you.." 1
Choice "Sorry, not now." 2

Good, here is the list of name of the recently deceased and the one whom you'll need to collect their remains from. While you're out in the field, surely you would be willing to deliver this letter to Musa Yahaz in Gnisis and ask her to sign it to confirm her registration with the remains donation.

Journal GG_Quest_2 20
Player->AddItem "GG_bk_recently_deceased" 1
Choice "Yes, sure." 3
Choice "Is this part of my assignment?" 4

Good, here is the... Wait, did you just say no? You need to understand that this is a pressing matter. Talk to me as soon as whatever task at hand is completed.
